app/style/list/start-ui.less

```diff
@@ -63,13 +63,20 @@
   overflow: hidden;
   flex: 1 1;
   flex-direction: column;
+  &.split-view {
```

Comment:
We usually have an empty line above new declarations

Reply:
👍